### PR TITLE
LEF-605: API listCollections

### PR DESCRIPTION
**Pull request by msalvatti**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-frontify@0.0.16-canary.207.27491bc.0
  npm install @friggframework/api-module-google-calendar@0.0.3-canary.207.27491bc.0
  npm install @friggframework/integrations@1.1.5-canary.207.27491bc.0
  npm install @friggframework/types@0.1.3-canary.207.27491bc.0
  # or 
  yarn add @friggframework/api-module-frontify@0.0.16-canary.207.27491bc.0
  yarn add @friggframework/api-module-google-calendar@0.0.3-canary.207.27491bc.0
  yarn add @friggframework/integrations@1.1.5-canary.207.27491bc.0
  yarn add @friggframework/types@0.1.3-canary.207.27491bc.0
  ```
</details>

### CODE REVIEW

**Review by leofmds — CHANGES_REQUESTED** [review]
LGTM, just a small change

File: api-module-library/frontify/api.js [hunk 1]
```diff
@@ -268,6 +268,23 @@ class Api extends OAuth2Requester {
         return response.data.brand.libraries.items;
     }
 
+    async listCollections(query) {
+      const ql = `query Collections {
+                    library(id: "${query.libraryId}") {
+                      collections {
+                        items {
+                          id
+                          name
```
[the remark this answers is not in the document]
Comment (msalvatti) [reply api-module-library/frontify/api.js:277]: ok, included.

**Review by leofmds — APPROVED** [review]
👍🏻

**Review by roboli — APPROVED** [review]

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-frontify@0.0.17`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-frontify%400.0.17) :rocket:

**Pull request merged**